packages/@ama-sdk/schematics/schematics/typescript/shell/index.spec.ts

```diff
@@ -95,4 +95,9 @@ describe('Typescript Shell Generator', () => {
     expect(npmTree.readContent('/package.json')).toContain('npm exec');
     expect(npmTree.readContent('/package.json')).not.toContain('yarn exec');
   });
+
+  it('should generate correct openapitools.json', () => {
+    const openApiTools = JSON.parse(yarnTree.readContent('/openapitools.json'));
+    expect('test-scope-test-sdk' in openApiTools['generator-cli'].generators).toBe(true);
```

Comment:
```suggestion
    expect(openApiTools['generator-cli'].generators).toEqual(expect.objectContaining({('test-scope-test-sdk': expect.anything()});
```
this helps in case of failure to have a relevant error instead of `expected true got false`